pages/citizen/registration.py

```diff
@@ -31,13 +35,27 @@ def profile(self):
         click(self.ID.btnProfile)
         return self
 
+    def error_message(self):
+        required = get(self.ID.lblRequired)
+        return required
+
+    def get_citizen_login_id(self):
+        return get(self.ID.lblMobileNumber)
+
+    def user_not_found(self):
+        return get(self.ID.lblUserNameValidation)
+
 
 @PageObject
 class OTPPage(Page):
     class ID:
         txtOTP = "input#otp"
         btnGetStarted = "button#otp-start"
         btnResend = "div#otp-resend"
+        lblOTP = "xpath=//label[@for='otp']/following-sibling::div[text()[1]]"
+        lblOTPSentTo = "xpath=//div[@class='label-text otp-mobile-number']"
```

Comment:
Use CSS here

Reply:
Not using this element on test case anymore.